src/sqllogictest/src/runner.rs

```diff
@@ -959,9 +959,24 @@ impl<'a> RunnerInner<'a> {
             |_, _| PubSubClientConnection::noop(),
         );
         let persist_clients = Arc::new(persist_clients);
+        let (tracing_handle, _tracing_guard) = config
```

Comment:
You may need to stash this `_tracing_guard` somewhere to keep it alive as long as the server's alive? Maybe it's better to create the tracing guard in the main function, and pass the handle in here? Idk! But that might be what's causing this not to work.

Reply:
Tried sticking it into a RefCell with no change.